Jenkinsfile

```diff
@@ -31,7 +31,7 @@
  * SUCH DAMAGE.
  */
 
-def XENADMIN_BRANDING_TAG = 'v4.0'
+def XENADMIN_BRANDING_TAG = 'v4.3'

```

Comment:
As discussed, better use a branch for the time being. Same below for the pipeline.